.github/workflows/create-image-without-release.yml

```diff
@@ -0,0 +1,172 @@
+# This workflow will release project with Gradle
+name: Create Image Only (No Release)
+
+on:
+    push:
+
+env:
+    IMAGE_BASE_DIR: ./containers
+
+jobs:
+    build:
```

Comment:
This job just runs in parallel with the `build-containers` job - so if you want to ensure the project builds, you need to put a `needs` block in `build-containers`. Or you could just assume the service builds properly and not have this `build` job at all.

Reply:
Done